cfme/utils/appliance/console.py

```diff
@@ -0,0 +1,428 @@
+import os
+import socket
+import tempfile
+from contextlib import contextmanager
+from re import escape as resc
```

Comment:
Personally I don't really like renaming this import, thoughts @ManageIQ/qe-committers?

Reply:
I don't like that it seems like some "rscue" would be done. I was not able to find out better name that would still be short for often use. Therefore I suggest using just `esc()` or just `r()`?

Reply:
`ree()` ?

Reply:
I would just `import re`, and call `re.escape`